app/models/teachers_pensions_service.rb

```diff
@@ -30,6 +31,36 @@ def self.recent_tps_school(claim)
 
     return nil unless tps_record&.school_urn
 
+    school_for_tps_record(tps_record)
+  end
+
+  def self.has_tps_school_for_student_loan_in_previous_financial_year?(claim)
+    tps_school_for_student_loan_in_previous_financial_year(claim).present?
+  end
+
+  def self.tps_school_for_student_loan_in_previous_financial_year(claim)
+    previous_academic_year = PolicyConfiguration.for(StudentLoans).current_academic_year - 1
+    start_of_previous_financial_year = Date.new(previous_academic_year.start_year, 4, 6)
+    end_of_previous_financial_year = Date.new(PolicyConfiguration.for(StudentLoans).current_academic_year.start_year, 4, 5)
```

Comment:
Suggestion: it would be useful to have these functions on the `AcademicYear` model.

Reply:
Stretch goal for another PR. Pressed for time.